charms/precise/clearwater-sipp/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+name: clearwater-sipp
+summary: SIPp charm for Project Clearwater
+maintainer: Project Clearwater Maintainers <maintainers@projectclearwater.org>
+description: SIPp charm for Project Clearwater
+tags:
+  - misc
+subordinate: false
+requires:
+  pcscf:
```

Comment:
Good point.  I don't have time to fix now but I think there's value in having this in without that, so I've raised an issue (https://github.com/Metaswitch/clearwater-juju/issues/17) to track it.